README.md

```diff
@@ -1,7 +1,81 @@
+| compute_merkle_root_depth_32 |                 7517 |            0.159 |            0.166 |          0.720 |
+
+`verify_proof` and `storage_proof_depth_8` were omitted as I ran into issues with v1.0.0-beta.0. Specifically.
+storage_proof_depth_8 depends on
+[aragonzkresearch/noir-trie-proofs](https://github.com/aragonzkresearch/noir-trie-proofs),
+which, at the time of writing, has not yet been updates for nargo
+v1.0.0-beta.0. Furthermore running `bb` to generate the recurvsive proof
+(`verify_proof`) emitted the following error:
+
+```
+bb: /home/runner/work/aztec-packages/aztec-packages/barretenberg/cpp/src/barretenberg/ecc/curves/bn254/../../fields/./field_declarations.hpp:128: bool bb::field<bb::Bn254FrParams>::operator bool() const [Params = bb::Bn254FrParams]: Assertion `(out.data[0] == 0 || out.data[0] == 1)' failed.
+Aborted (core dumped)
+```
+
+| Backend Circuit Size | Compile Time (s) | Execute Time (s) |        Prove Time (s)       |
+|:--------------------:|:----------------:|:----------------:|:---------------------------:|
+| 2^2                  |            0.162 |            0.148 |                     0.143   |
+| 2^3                  |            0.146 |            0.146 |                     0.143   |
+| 2^4                  |            0.149 |            0.151 |                     0.145   |
+| 2^5                  |            0.146 |            0.146 |                     0.147   |
+| 2^6                  |            0.154 |            0.146 |                     0.148   |
+| 2^7                  |            0.156 |            0.150 |                     0.154   |
+| 2^8                  |            0.143 |            0.154 |                     0.160   |
+| 2^9                  |            0.144 |            0.161 |                     0.166   |
+| 2^10                 |            0.156 |            0.157 |                     0.179   |
+| 2^11                 |            0.165 |            0.181 |                     0.205   |
+| 2^12                 |            0.172 |            0.203 |                     0.248   |
+| 2^13                 |            0.207 |            0.269 |                     0.364   |
+| 2^14                 |            0.269 |            0.415 |                     0.541   |
+| 2^15                 |            0.421 |            0.693 |                     0.812   |
+| 2^16                 |            0.714 |            1.252 |                     1.444   |
+| 2^17                 |            1.251 |            2.355 |                     2.690   |
+| 2^18                 |            2.562 |            4.797 |                     5.114   |
+| 2^19                 |            5.122 |            9.672 |                     10.080  |
+| 2^20                 |           10.416 |           19.475 |                     19.853  |
+| 2^21                 |           20.047 |           37.673 |                     302.956 |
```

Comment:
The proving times of 2^21 and 2^22 look weirdly long compared to the results before them 🤔

Reply:
Addressed in https://github.com/Savio-Sou/noir-benchmarks/pull/6 :)